cypress/integration/pages/testsForAllPages.js

```diff
@@ -318,6 +318,14 @@ export const testsThatFollowSmokeTestConfigforAllPages = ({
           .should('have.attr', 'href', '#content');
       });
 
+      if (variant !== 'default') {
```

Comment:
I think I'll move this condition inside and check that there _isn't_ a script switch when there shouldn't be as well